pandas/core/indexes/datetimes.py

```diff
@@ -2040,7 +2040,7 @@ def date(self):
         Returns numpy array of python datetime.date objects (namely, the date
         part of Timestamps without timezone information).
         """
-        return libts.ints_to_pydatetime(self.normalize().asi8, box="date")
+        return libts.ints_to_pydatetime(self.asi8, box="date")
```

Comment:
I _think_ this needs to be `self._local_timestamps()` instead of `self.asi8` since those are essentially epoch (UTC) timestamps and we want to return local values.

Reply:
Thanks for the suggestion. That might be it.